scale/trigger/models.py

```diff
@@ -144,7 +144,8 @@ def get_by_natural_key(self, name):
 
         return self.get(name=name)
 
-
+# TODO 1181: Modify usage when remove triggers in v6
+# Scale Clock utilizes trigger rules
 class TriggerRule(models.Model):
```

Comment:
how tricky would it be to have something else create trigger events for the clock?  If we're going to leave it we should probably remove the TODO and explain that the trigger rule is being left behind only for the clock.